lib/v2/jasa/latest.js

```diff
@@ -0,0 +1,74 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const path = require('path');
+const { art } = require('@/utils/render');
+
+module.exports = async (ctx) => {
+    const resp = await got('https://asa.scitation.org');
+    const cookie = resp.headers['set-cookie'].join(' ');
+    const hostUrl = 'https://asa.scitation.org/toc/jas/current?size=all';
+
+    const curtResponse = await got({
+        method: 'get',
+        url: hostUrl,
+        headers: {
+            'Accept-Encoding': 'gzip, deflate, br',
+            'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/94.0.4606.81 Safari/537.36',
```

Comment:
https://github.com/DIYgod/RSSHub/blob/a43b67bc1f6d04519c09ac04f3fce4c9e074381a/lib/config.js#L40

`UA` should be configured via env instead of hard-coded.

Reply:
Thanks your attention, I have modified it.